docs/getting-started.md

```diff
@@ -4,25 +4,39 @@ sidebar_position: 1
 
 # Getting Started
 
-The `@lukso/lsp6-signer.js` package is used to sign an LSP6 Execute Relay Call transaction.
+- [GitHub Repository](https://github.com/lukso-network/tools-lsp6-signer)
+- [NPM Package](https://www.npmjs.com/package/@lukso/lsp6-signer.js)
 
-This library will add the `\x19Execute Relay Call:\n` prefix to a message and sign it.
+The `@lukso/eip191-signer.js` package is used to sign any EIP191 data.
 
-The ExcuteRelayCall prefix is used instead of the standard Ethereum transaction prefix to sign messages so that an executeRelayCall transaction cannot be inadvertently signed when signing an Ethereum signed message.
+The following format is used to sign data :
 
-- [GitHub Repository](https://github.com/lukso-network/tools-lsp6-signer)
-- [NPM Package](https://www.npmjs.com/package/@lukso/lsp6-signer.js)
+```bash
+0x19 <1 byte version> <version specific data> <data to sign>
+```
+
+In the case of an Ethereum Signed Message :
+
+- 1 byte version = `0x45`
+- version specific data = `thereum Signed Message:\n + len(message)`
+
+In the case of data intended to validator :
```

Comment:
```suggestion
In the case of data intended to validator:
```

intended to validator? Idk it sounds weird

Reply:
In the case of data with intended validator: